src/k-provider/ovp/index.js

```diff
@@ -2,11 +2,12 @@
 import OVPProvider from './provider';
 import RequestBuilder from '../../util/request-builder';
 import * as ResponseTypes from './response-types';
+import {OVPConfiguration} from './config';
 
 declare var __VERSION__: string;
 declare var __NAME__: string;
 
 const NAME = __NAME__ + '-ovp';
 const VERSION = __VERSION__;
 
-export {OVPProvider as Provider, NAME, VERSION, RequestBuilder, ResponseTypes};
+export {OVPProvider as Provider, NAME, VERSION, RequestBuilder, ResponseTypes, OVPConfiguration};
```

Comment:
```suggestion
export {OVPProvider as Provider, NAME, VERSION, RequestBuilder, ResponseTypes, OVPConfiguration as DefaultConfiguration};
```

Reply:
@RoyBregman Maybe  **OVPDefaultConfiguration** ? to highlight that is relevant only for OVP ?

Reply:
not needed. the player is already typed. there is _kaltura-**ovp**-player_ and _kaltura-**tv**-player_